javascript/testing_javascript/more_testing.md

```diff
@@ -67,13 +67,12 @@ There are two solutions to the 'tightly coupled code' problem.  The first, and b
 
 <div class="lesson-content__panel" markdown="1">
 
-1. If you haven’t already, watch [FunFunFunction's "Mocking Basics" video](https://www.youtube.com/watch?v=3PjdxjWK0F0).
+1. If you haven’t already, watch [about mocking basics](https://www.youtube.com/watch?v=ajiAl5UNzBU&t=3024s).
```

Comment:
Little reword of the assignment step, as the "If you haven't already" bit in particular is a little odd with the new link.
```suggestion
1. Watch this [video about mocks in testing](https://www.youtube.com/watch?v=ajiAl5UNzBU&t=3024s).
```